docs/products/kafka/howto/kcat.rst

```diff
@@ -0,0 +1,112 @@
+Use kcat with Aiven for Apache Kafka
+====================================
+
+`kcat <https://github.com/edenhill/kcat>`_ (formerly known as kafkacat) is a generic non-JVM producer and consumer for Apache Kafka. 
+
+It can be used to produce and consume records to Apache Kafka topics as well as to list service configurations.
+
+
+Install kcat
+-------------
+
+kcat is an open source tool available from GitHub at https://github.com/edenhill/kcat. Installation instructions are provided in the repository README.
+
+Retrieve Aiven for Kafka SSL certificate files
+----------------------------------------------
+
+Aiven for Apache Kafka by default enables TLS security. 
+The certificates can be manually downloaded from the service overview page in the Aiven console, or via the :ref:`dedicated Aiven CLI command <avn_service_user_creds_download>`.
+
+Setup a kcat configuration file
+-------------------------------
+
+While kcat accepts all connection configuration parameters in the command line, using a configuration file helps minimising the code needed for any sequent calls.
+
+.. Note:: 
+
+    This example requires kcat **version 1.4.0-RC1** or greater (the ``-F`` option was added after 1.3.1). You can specify the same options on the command line with earlier versions.
```

Comment:
"You can specify the same options on the command line with earlier versions." - does it mean it is backward compatible? I'm not exactly sure what the phrase mean

Reply:
I'm going to remove this since the version 1.4.0 is already 2 years old... We can assume everyone is on the new release?

Reply:
What do you think?

Reply:
Yes, I was thinking that this phrase doesn't age well. I'm fine with removing it